### PR TITLE
[C-1660] Clear feed on sign-out

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Clears feed on sign out so new signed-in user get's their feed.
- Refactors AsyncStorage keys into one file to make it easier to understand storage
- Uses this file to aid in clearing storage keys on sign-out

### CODE REVIEW

File: packages/mobile/src/store/account/sagas.ts [hunk 1]
```diff
@@ -2,11 +2,11 @@ import { accountActions, getContext } from '@audius/common'
...
 import { updateProfileAsync } from 'common/store/profile/sagas'
 import { takeEvery, call } from 'typed-redux-saga'
+
+import { IS_MOBILE_USER } from 'app/constants/storage-keys'
 const { signedIn } = accountActions
 
-export const RESET_REQUIRED_KEY = 'password-reset-required'
-export const ENTROPY_KEY = 'hedgehog-entropy-key'
-export const IS_MOBILE_USER_KEY = 'is-mobile-user'
+// const RESET_REQUIRED_KEY = 'password-reset-required'
```
Comment (nicoback2) [inline packages/mobile/src/store/account/sagas.ts:9]: delete?

File: packages/mobile/src/constants/storage-keys.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export const DEVICE_TOKEN = '@device-token'
```
Comment (sliptype) [inline packages/mobile/src/constants/storage-keys.ts:1]: Love the consistency here 😂

File: packages/mobile/src/store/account/sagas.ts (hunk 1 above)
Comment (sliptype) [inline packages/mobile/src/store/account/sagas.ts:9]: Is this not being used at all?

**Review by sliptype — APPROVED** [review]
This is way cleaner!

File: packages/mobile/src/constants/storage-keys.ts (hunk 2 above)
Comment (dylanjeffers) [reply packages/mobile/src/constants/storage-keys.ts:1]: It's sad too cause we can't really change them, lol

File: packages/mobile/src/store/account/sagas.ts (hunk 1 above)
Comment (dylanjeffers) [reply packages/mobile/src/store/account/sagas.ts:9]: So there's a commented out section below it that used it, so I kept this around so when we do work on that, we have the key avail?

**Review by nicoback2 — APPROVED** [review]

**Pull request merged**